### ISSUE TITLE
Installationd dependency on Ubuntu

### ISSUE THREAD

**Issue by heeren-cliqz** [cliqz-oss/keyvi#90]
Apart from the installations steps mentioned,
one must need to install few things as well, otherwise one get msgrpc package not found and other similar exceptions. 
`easy_install` msgpack-python
apt-get install python-tk python-setuptools
sudo python -m easy_install -U snappy`
these are the requirements to download this on c3.2xlarge machine on AWS.

**Comment by hendrik-cliqz** [comment]
You are right, msgpack-python and python-snappy are missing in the package dependencies of the ubuntu package. 

But I do not get those: python-tk python-setuptools These are not used by python-keyvi.

I will fix it in the next packages.

**Comment by hendrik-cliqz** [comment]
the dependency on python-snappy got removed.
